internal/receiver/databricksreceiver/documentation.md

```diff
@@ -8,18 +8,175 @@ These are the metrics available for this scraper.
 
 | Name | Description | Unit | Type | Attributes |
 | ---- | ----------- | ---- | ---- | ---------- |
-| databricks.jobs.active.total | A snapshot of the number of active jobs taken at each scrape | {jobs} | Gauge(Int) | <ul> </ul> |
-| databricks.jobs.run.duration | The execution duration in milliseconds per completed job | ms | Gauge(Int) | <ul> <li>job_id</li> </ul> |
-| databricks.jobs.schedule.status | A snapshot of the pause/run status per job taken at each scrape 0=PAUSED, 1=UNPAUSED, 2=NOT_SCHEDULED  | {status} | Gauge(Int) | <ul> <li>job_id</li> </ul> |
-| databricks.jobs.total | A snapshot of the total number of jobs registered in the Databricks instance taken at each scrape | {jobs} | Gauge(Int) | <ul> </ul> |
-| databricks.tasks.run.duration | The execution duration in milliseconds per completed task | ms | Gauge(Int) | <ul> <li>job_id</li> <li>task_id</li> </ul> |
-| databricks.tasks.schedule.status | A snapshot of the pause/run status per task taken at each scrape 0=PAUSED, 1=UNPAUSED, 2=NOT_SCHEDULED  | {status} | Gauge(Int) | <ul> <li>job_id</li> <li>task_id</li> <li>task_type</li> </ul> |
-
-## Attributes
-
-| Name | Description |
-| ---- | ----------- |
-| databricks.instance.name | The name of the Databricks instance as defined by the value of the "instance_name" field in the config |
-| job_id | The numeric ID of the Databricks job |
-| task_id | The name of the Databricks task |
-| task_type | The type of the Databricks task |
+| **databricks.jobs.active.total** | A snapshot of the number of active jobs taken at each scrape | {jobs} | Gauge(Int) | <ul> </ul> |
+| **databricks.jobs.run.duration** | The execution duration in milliseconds per completed job | ms | Gauge(Int) | <ul> <li>job_id</li> </ul> |
```

Comment:
not introduced here but is this is a constant value (or even a single datapoint) or are job ids consistent through the lifetime of the ~workflow?

Reply:
Jobs are often run on an interval and they retain the job id throughout, so the idea is to let users see how their jobs are performing over time.